intranet/static/js/common.nav.js

```diff
@@ -0,0 +1,39 @@
+$(function() {
```

Comment:
Sure, can you add the integration using @tjhsst-ion though?
I'll work on fixing codacy.